index.bs

```diff
@@ -3252,7 +3275,8 @@ PublicKeyCredential.signalCurrentUserDetails({
     rpId: "example.com",
     userId: "aabbcc",  // user handle, base64url.
     name: "New user name",
-    displayName: "New display name"
+    displayName: "New display name",
+    language: "en",
```

Comment:
You might wish to include direction here also, as an example. In this case, the direction would be `ltr`

Reply:
I considered it but decided to omit `direction` to indicate that the fields are optional, but fair enough. We can indicate that with comments instead.